enterprise/frontend/src/embedding-sdk/index.ts

```diff
@@ -25,4 +25,4 @@ export type {
   MetabaseComponentTheme,
 } from "./types/theme";
 
-export type { Dashboard } from "metabase-types/api";
+export type { Dashboard as MetabaseDashboard } from "metabase-types/api";
```

Comment:
I couldn't find where this is used yet. Are you planning to use this in later PRs?